src/liquid/components/ld-input/test/ld-input.spec.ts

```diff
@@ -61,52 +61,47 @@ describe('ld-input', () => {
     expect(ldInput).toMatchSnapshot()
   })
 
-  it('emits focus and blur event', async () => {
+  xit('emits focus and blur event', async () => {
```

Comment:
Let's leave a comment why we `xit`ed this one.

Reply:
```suggestion
  // TODO: Uncomment, as soon as Stencil's JSDom implementation
  // supports bubbling of composed events into the light DOM.
  xit('emits focus and blur event', async () => {
```